packages/next/server/web/next-url.ts

```diff
@@ -61,6 +62,16 @@ export class NextURL {
   private analyzeUrl() {
     const { headers = {}, basePath, i18n } = this[Internal].options
 
+    if (this[Internal].url.pathname.startsWith('/_next/data/')) {
```

Comment:
Similar here, do we need to handle `basePath` or is it already removed by this point?